server/src/main/java/org/elasticsearch/cluster/coordination/StableMasterHealthIndicatorService.java

```diff
@@ -219,9 +243,9 @@ private String getNameForNodeId(String nodeId) {
      * @param explain If true, the returned list includes a UserAction to contact support, otherwise an empty list
```

Comment:
I can't comment on the line above this one, but this javadoc is now incorrect, can you fix it?

Reply:
++ thanks, fixed in 7b1ea00b374.